jupyter-book/cellular_structure/integration.ipynb

```diff
@@ -1949,15 +2022,15 @@
    "id": "40fd2b7d",
    "metadata": {},
    "source": [
-    "## Mutual nearest neighbors based"
+    "## Linear embedding models"
```

Comment:
There is a capitalisation missing in "to correct the batch effect. mutual nearest neighbors are pairs of cells from two different "

Reply:
closed via [ced74ec](https://github.com/theislab/single-cell-best-practices/pull/87/commits/ced74ec1d4314711ae2f5d16b0bc6d35eb6eaaa7)